openshift_tools/monitoring/zagg_sender.py

```diff
@@ -27,6 +27,9 @@
 import os
 import yaml
 
+# Add Hawkular sender backend
+from openshift_tools.monitoring.hawk_sender import HawkSender
```

Comment:
I really don't want zagg_sender to call hawk_sender.

I think we need a meta sender class, possible called `metric_sender` that then sends to both `zagg_sender` and `hawk_sender`.

I know that this will mean switching the collectors to call into `metric_sender` and that's a bigger change, but I think it's needed.